doc/components_2.md

```diff
@@ -24,8 +24,8 @@ tags to make it easier to preview outside the main template
 </mjml>
 ```
 
-The `MJML` engine will then replace your included files before starting the rendering process
+The `MJML` engine will then replace your included files before starting the rendering process.
 
 <aside class="notice">
```

Comment:
This is not true anymore so we can remove it

Reply:
Done.